src/Framework/ContaoFramework.php

```diff
@@ -243,10 +243,14 @@ private function getRoute()
             return 'console';
         }
 
-        $route = $this->router->generate(
-            $this->request->attributes->get('_route'),
-            $this->request->attributes->get('_route_params')
-        );
+        try {
+            $route = $this->router->generate(
+                $this->request->attributes->get('_route'),
+                $this->request->attributes->get('_route_params')
+            );
+        } catch (\InvalidArgumentException $e) {
+            return null;
```

Comment:
I also doubt that anyone uses `TL_SCRIPT` an the command line, but if you are worried about BC, we can just keep the return value.